modules/byo-vpc-prep.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * osd_cluster_create/managing-byo-vpc-single-az.adoc
+// * osd_cluster_create/managing-byo-vpc-multiple-az.adoc
+
+[id="byo-vpc-prep_{context}"]
+= Preparing your VPC to use with {product-title}
+
+Using your own VPC requires an AWS account, a configured VPC, and an attached internet gateway.
+
+.Procedure
+
+. Log in to your account on the link:https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_create.html[AWS Management Console].
+. Create a VPC with the IP address of `10.0.0.0/16` and enable `dnsHostnames`. For more information, see the link:https://docs.aws.amazon.com/vpc/latest/userguide/vpc-dns.html#vpc-dns-support[supported DNS values for the VPC in AWS].
+. Create an internet gateway, and attach it to the VPC created in the last step.
```

Comment:
'in the last step' isn't clear. 
Perhaps:
* in the previous step
* the VPC you just created.